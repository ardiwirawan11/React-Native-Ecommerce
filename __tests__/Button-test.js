import React from 'react';
import { shallow } from 'enzyme';
import Button from '../src/components/Button';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });
describe('Button', () => {
    describe('Rendering', () => {
        it('should match to snapshot', () => {
            const component = shallow(<Button/>);
            expect(component).toMatchSnapshot();
        });
    });
});