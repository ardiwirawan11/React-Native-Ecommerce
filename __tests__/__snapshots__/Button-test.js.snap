// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Rendering should match to snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  style={
    Object {
      "alignSelf": "stretch",
      "backgroundColor": "#38ba7d",
      "boderWidth": 1,
      "borderButtomColor": "#1e6343",
      "borderButtomWIdth": 6,
      "height": 45,
      "justifyContent": "center",
      "marginLeft": 15,
      "marginRight": 15,
    }
  }
>
  <Text
    style={
      Object {
        "alignSelf": "center",
        "color": "#fff",
        "fonsSize": 16,
        "fontWeight": "600",
      }
    }
  >
     textInComponent 
  </Text>
</TouchableOpacity>
`;
